util/llpcElfReader.cpp

```diff
@@ -181,7 +183,7 @@ Result ElfReader<Elf>::GetSectionData(
 // =====================================================================================================================
 // Gets the count of symbols in the symbol table section.
 template<class Elf>
-uint32_t ElfReader<Elf>::GetSymbolCount()
+uint32_t ElfReader<Elf>::GetSymbolCount() const
```

Comment:
Could you please split the const-correctness changes out into their own patch? Then we should be able to get those out of the way trivially.

Reply:
I've opened https://github.com/GPUOpen-Drivers/llpc/pull/439 for the const correctness work.  I'll rebase on top of that once it is merged.